sklearn/naive_bayes.py

```diff
@@ -115,6 +115,16 @@ class GaussianNB(BaseNB):
 
     Read more in the :ref:`User Guide <gaussian_naive_bayes>`.
 
+    Parameters
+    ----------
+    fit_prior : boolean
+        Whether to learn class prior probabilities or not.
+        If false, a uniform prior will be used.
+
+    class_prior : array-like, size=(n_classes,)
+        Prior probabilities of the classes. If specified the priors are not
+        adjusted according to the data.
```

Comment:
I just check the code the API of LDA. There is a slight difference:
- in LDA, prior will be either set by the user or automatically using the sample distribution.
- in NB, prior will be either set by the user, automatically using the sample distribution (fit_prior=True), or automatically using the number of class (fit_prior=False).

Shall I remove the case that class prior is set considering the number of class or shall I not follow the same API?